packages/editor/src/Editor.tsx

```diff
@@ -867,7 +877,10 @@ const EditorContent = ({
     });
   }, []);
 
+  const [canvasLoaded, setCanvasLoaded] = useState(false);
+
   useEffect(() => {
+    sendCanvasData();
     if (window.editorWindowAPI.onUpdate) {
       window.editorWindowAPI.onUpdate();
```

Comment:
Do we need this call if we have `sendCanvasData`?

Reply:
Yes, we need to sendCanvasData every time data is updated!